tests/basic_relay.go

```diff
@@ -0,0 +1,298 @@
+	)
+
+	relayerConfig := config.Config{
+		LogLevel:          logging.Info.LowerString(),
+		NetworkID:         peers.LocalNetworkID,
+		PChainAPIURL:      subnetAInfo.ChainNodeURIs[0],
+		EncryptConnection: false,
+		StorageLocation:   storageLocation,
+		SourceSubnets: []config.SourceSubnet{
+			{
+				SubnetID:          subnetAInfo.SubnetID.String(),
+				ChainID:           subnetAInfo.BlockchainID.String(),
+				VM:                config.EVM.String(),
+				EncryptConnection: false,
+				APINodeHost:       hostA,
+				APINodePort:       portA,
+				MessageContracts: map[string]config.MessageProtocolConfig{
+					teleporterContractAddress.Hex(): {
+						MessageFormat: config.TELEPORTER.String(),
+						Settings: map[string]interface{}{
+							"reward-address": fundedAddress.Hex(),
+						},
+					},
+				},
+			},
+		},
+		DestinationSubnets: []config.DestinationSubnet{
+			{
+				SubnetID:          subnetBInfo.SubnetID.String(),
+				ChainID:           subnetBInfo.BlockchainID.String(),
+				VM:                config.EVM.String(),
+				EncryptConnection: false,
+				APINodeHost:       hostB,
+				APINodePort:       portB,
+				AccountPrivateKey: hex.EncodeToString(fundedKey.D.Bytes()),
+			},
+		},
+	}
+
+	data, err := json.MarshalIndent(relayerConfig, "", "\t")
+	Expect(err).Should(BeNil())
+
+	f, err := os.CreateTemp(os.TempDir(), "relayer-config.json")
+	Expect(err).Should(BeNil())
+
+	_, err = f.Write(data)
+	Expect(err).Should(BeNil())
+	relayerConfigPath := f.Name()
+
+	log.Info("Created awm-relayer config", "configPath", relayerConfigPath, "config", string(data))
+
+	//
+	// Build Relayer
+	//
+	// Build the awm-relayer binary
+	cmd := exec.Command("./scripts/build.sh")
+	out, err := cmd.CombinedOutput()
+	fmt.Println(string(out))
+	Expect(err).Should(BeNil())
+
+	//
+	// Send a transaction to Subnet A to issue a Warp Message from the Teleporter contract to Subnet B
+	//
+	ctx := context.Background()
+
+	relayerCmd, relayerCancel = testUtils.RunRelayerExecutable(ctx, relayerConfigPath)
+
+	nonceA, err := subnetAInfo.ChainWSClient.NonceAt(ctx, fundedAddress, nil)
+	Expect(err).Should(BeNil())
+
+	nonceB, err := subnetBInfo.ChainWSClient.NonceAt(ctx, fundedAddress, nil)
+	Expect(err).Should(BeNil())
+
+	log.Info("Packing teleporter message", "nonceA", nonceA, "nonceB", nonceB)
+	payload, err = teleporter.PackSendCrossChainMessageEvent(common.Hash(subnetBInfo.BlockchainID), teleporterMessage)
+	Expect(err).Should(BeNil())
+
+	data, err = teleporter.EVMTeleporterContractABI.Pack(
+		"sendCrossChainMessage",
+		teleporterTestUtils.SendCrossChainMessageInput{
+			DestinationChainID: subnetBInfo.BlockchainID,
+			DestinationAddress: fundedAddress,
+			FeeInfo: teleporterTestUtils.FeeInfo{
+				ContractAddress: fundedAddress,
+				Amount:          big.NewInt(0),
+			},
+			RequiredGasLimit:        big.NewInt(1),
+			AllowedRelayerAddresses: []common.Address{},
+			Message:                 []byte{1, 2, 3, 4},
```

Comment:
can you use the `teleporterMessage` fields so they still in unison i.e. `teleporterMessage.Message`, etc.